src/App.tsx

```diff
@@ -3,9 +3,64 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
+import { getMovie } from './api';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const [newMovie, setNewMovie] = useState<Movie | null>(null);
+  const [newTitle, setNewTitle] = useState('');
+  const [isMovieFind, setIsMovieFind] = useState(true);
+  const [isError, setIsError] = useState(false);
+  const [isFirstSearch, setIsFirstSearch] = useState(true);
+
+  const changeTitle = (event:React.ChangeEvent<HTMLInputElement>) => {
+    setNewTitle(event.target.value);
+    setIsError(false);
+  };
+
+  const findMovie = (event:React.MouseEvent<HTMLButtonElement>) => {
+    event.preventDefault();
+    setIsMovieFind(false);
+
+    getMovie(newTitle)
+      .then(res => {
+        if ('Error' in res) {
+          setNewMovie(null);
+          setIsMovieFind(true);
+          setIsError(true);
+        } else {
+          setNewMovie({
+            title: newTitle,
```

Comment:
You don't need to use a searched text as a title. Use a title from the response as is. And search text you should use only to find the movie by its title or description.